### PR TITLE
Make sure special accounts are not returned in results 

### PR DESCRIPTION
**Pull request by algorotem**
## Summary

Solves #310 

## Test Plan
integration test for both endpoints

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -706,3 +726,18 @@ func max(x, y uint64) uint64 {
 	}
 	return y
 }
+
+// isSpecialAccount returns true if addr belongs to a special account, false otherwise.
+// The function returns an error in case it fails to retrieve the special accounts list.
+func (si *ServerImplementation) isSpecialAccount(addr string) (bool, error) {
+	// Special accounts non handling
+	sa, err := si.db.GetSpecialAccounts()
```
Comment (brianolson) [inline api/handlers.go:734]: probably worth caching this in a field of ServerImplementation so we don't hit the db every time.

Comment (algorotem) [reply api/handlers.go:734]: I thought about it but the db implementation already caches it, should we cash it twice?

Comment (brianolson) [reply api/handlers.go:734]: Oh oops, yeah postgres.go caching is enough

**Review by brianolson — APPROVED** [review]

**Review by winder — APPROVED** [review]
LGTM

**Pull request merged**